package.json

```diff
@@ -26,7 +26,7 @@
     "dev": "npm run build -- --watch",
     "example:dev": "cp README.md example/README.md && npm -C example run dev",
     "example:build": "cp README.md example/README.md && npm -C example run build",
-    "build": "tsup src/index.ts --dts --format cjs,esm",
+    "build": "tsup src/index.ts --dts --external @vue/compiler-sfc --format cjs,esm",
```

Comment:
rewirth transformation with`@vue/compiler-sfc`? what do you think

Reply:
tsup should exclude depends in peerDeps already

Reply:
yeah, it's in `devDep` now
if rewrite with it, we should move it into `dep`

Reply:
Umm, seems like I forgot to add it. Let's go with peerDeps, thanks!